recipe/meta.yaml

```diff
@@ -1,31 +1,33 @@
 {% set name = "skein" %}
-{% set version = "0.8.0" %}
+{% set version = "0.8.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
-  sha256: 71da4a16e2bba4d99fa90454b58e951463452596a6e6afaaf024a0d1cce87d3f
+  url: https://files.pythonhosted.org/packages/py2.py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py2.py3-none-any.whl
+  sha256: 506884a13675ec884b1bbd85675f196fe123e13f33f6b308303531cd26466726
 
 build:
-  number: 1
-  skip: True  # [win or py<35]
-  script: python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-py3-none-any.whl
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-py2.py3-none-any.whl
   entry_points:
     - skein = skein.cli:main
 
 requirements:
   host:
-    - python
+    - python >=3.5
     - pip
   run:
-    - python
+    - python >=3.5
     - cryptography
     - grpcio >=1.11.0
     - protobuf >=3.5.0
     - pyyaml
+    # Require the unix virtual package as a runtime dep
+    - __unix
```

Comment:
This is a relatively new feature in Conda 4.9.0 included with PR ( https://github.com/conda/conda/pull/10214 ), which let's us specify OS constraints (like requiring Unix). As a result we can make this `noarch`. Though can revert this if we are concerned about trying a feature that requires a fairly new Conda version

Reply:
How "fairly new"? I don't particularly care how this package is built, provided most users that need it can install it. If this requires users use a conda version released in the last 6 months I think we'll want to hold off on this, not everyone upgrades quickly.

Reply:
Yep makes sense. Will revert these changes as this came out in mid-October 2020

Reply:
Ok this is now reverted